action.yml

```diff
@@ -79,6 +83,10 @@ runs:
       with:
         files: "${{ inputs.dockerfile }}"
         allow_failure: true # fails the Action if Dockerfile is missing
+        
+
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v2
```

Comment:
If the installation of qemu is as simple as calling 'apt install qemu-user-static' I would rather install it manually then calling some other action and thus adding new dependency.

It is true that the action would be then fixed only for debian-based systems, however I do not see problem in it because the only Linux runner that GitHub provides is Ubuntu.
The step should also be conditional.

Reply:
Apt install qemu-static was giving issue of glibc error CPU lacks VXE support (z14 or later required). To overcome that i have used qemu action which is working fine. I have shared the github action link below of succesful execution 
https://github.com/modassarrana89/s2i-base-container/actions/runs/5011219036.

Reply:
Thanks for explanation. In that case this seems reasonable.